plugins/modules/ntnx_promote_protected_resources_v2.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Nutanix
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: ntnx_promote_protected_resources_v2
+short_description: Module to promote a protected resource in Nutanix Prism Central.
+description:
+  - This module can be used to promote a protected resource in Nutanix Prism Central.
+  - Supported only for protected resources that have synchronous protection policies.
+options:
+  ext_id:
+    description:
+      - The external identifier of a protected VM or volume group.
+    type: str
+    required: true
+  wait:
+    description:
+      - Wait for the task to complete.
+    type: bool
+    required: false
+    default: True
+extends_documentation_fragment:
+  - nutanix.ncp.ntnx_credentials
+  - nutanix.ncp.ntnx_operations_v2
+author:
+  - George Ghawali (@george-ghawali)
+"""
+
+EXAMPLES = r"""
+- name: Promote VM
+  nutanix.ncp.ntnx_promote_protected_resources_v2:
+    nutanix_host: "{{ availability_zone_pc_ip }}"
```

Comment:
This module is used to replicate a promote a protected resource. Why it will be run against target PC ? Shouldnt this be run with PC IP where VM is located ?

Reply:
```suggestion
    nutanix_host: "<secondary-site-pc-ip>"
```